src/pages/index.js

```diff
@@ -0,0 +1,348 @@
+const deleteForm = deleteModal.querySelector(".modal__form");
+const deleteModalCloseBtn = deleteModal.querySelector(".modal__close-btn");
+
+//Avatar Modal Elements
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarSubmitBtn = avatarModal.querySelector(".modal__submit-btn");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+const avatarCloseBtn = avatarModal.querySelector(".modal__close-btn");
+
+// Preview Image Popup Elements
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageEl = previewModal.querySelector(".modal__image");
+const previewModalCaptionEl = previewModal.querySelector(".modal__caption");
+const previewModalCloseButton = previewModal.querySelector(".modal__close-btn");
+const modals = document.querySelectorAll(".modal");
+
+const editProfileModal = document.querySelector("#edit-profile-modal");
+const editFormElement = editProfileModal.querySelector(".modal__form");
+const editModalCloseButton = document.querySelector(".modal__close-btn");
+const editModalNameInput = editProfileModal.querySelector(
+    "#profile-name-input"
+);
+const editModalDescriptionInput = editProfileModal.querySelector(
+    "#profile-description-input"
+);
+
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+
+let selectedCard, selectedCardId;
+
+function handleEscape(evt) {
+    if (evt.key === "Escape") {
+        const activeModal = document.querySelector(".modal_opened");
+        closeModal(activeModal);
+    }
+}
+
+modals.forEach((modal) => {
+    modal.addEventListener("mousedown", (evt) => {
+        if (evt.target.classList.contains("modal")) {
+            closeModal(modal);
+        }
+    });
+});
+
+function getCardElement(data) {
+    const cardElement = cardTemplate.content
+        .querySelector(".card")
+        .cloneNode(true);
+
+    //Card Elements
+    const cardNameEl = cardElement.querySelector(".card__title");
+    const cardImageEl = cardElement.querySelector(".card__image");
+    const cardLikeBtn = cardElement.querySelector(".card__like-button");
+    const cardDeleteBtn = cardElement.querySelector(".card__delete-icon");
+
+    //TODO - if the card is liked set the active class on the card
+
+    cardNameEl.textContent = data.name;
+    cardImageEl.src = data.link;
+    cardImageEl.alt = data.name;
+
+    function handleLike(evt, id) {
+        // remove - evt.target.classList.toggle("card__like-button_liked");
+        // 1. check weather the card is liked or not
+        // const isLiked=?
+        // 2.call the handkeLikeStatus method,passing the appropriate arguements
+        // 3.handle the response (.then and .catch)
+        // 4.in the .then toggle the active class
+    }
+
```

Comment:
* When I reload the page, all **red** hearts become **transparent** again.

Please, check `isLiked` here